src/20240324_AaveV3Ethereum_FundingUpdatePartB/FundingUpdatePartB.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Funding Update (Part B)"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-funding-update/16675/4"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x4dd4dff7096bf7ab8c4c071975d40f4cf709c41b4b6b7c60777a6dd50d2ecd09"
+---
+
+## Simple Summary
+
+This publication mainly proposes consolidating DAO assets in preparation of foreseeable DAO expenses. There are additional small treasury management actions included in this proposal for house-keeping.
+
+## Motivation
```

Comment:
This needs updated text as in this proposal there is no bridging from polygon etc